### DISCUSSION TITLE
How to create a Brunch plugin which will output two files?

### DISCUSSION THREAD

**Discussion by ghost** [brunch/brunch#1880] [Q&A]
I need to convert files with the same extension (*.moon) into two files - _\*.js_ and _\*.css_.
How can I do it?

**Comment by shvaikalesh** [comment]
Hey @zakharkang, thanks for the issue.

Currently, Brunch does not allow returning multiple files from any of the API methods. However, you can anything you have to in `onCompile` hook: please see how [vue-brunch](https://github.com/theocodes/vue-brunch/blob/master/source/index.js) extracts CSS.

We are investigating use cases and completing refactoring and probably allow returning (async) iterable of files in future.

**Comment by ghost** [comment]
Thanks a lot! I was confused by "onCompileStatic" function in the docs.
Then... Is it possible to get the Brunch build type (production, for example)?
